Jenkinsfile

```diff
@@ -25,9 +25,6 @@ def rocmtestnode(Map conf) {
             export CXX=${compiler}
             export CXXFLAGS='-Werror'
             env
-            whoami
-            echo $UID
-            ls -la /root
```

Comment:
I added them in there for debug.  The build failed prior to me seeing the output and I found the details in a different way.  I saw that in the Jenkins invocation it would have shown UID == 1001.